front/lib/api/assistant/configuration.ts

```diff
@@ -865,6 +870,42 @@ export async function createAgentConfiguration(
             transaction: t,
           }
         );
+
+        if (created) {
+          const user = auth.getNonNullableUser();
+          // Create a default group for the agent and add the author to it.
+          const defaultGroup = await GroupResource.makeNew({
+            workspaceId: owner.id,
+            name: `Group for Agent ${agentConfigurationInstance.name}`,
+            kind: "regular",
+          });
+
+          // Add user to the newly created group
+          const addMemberResult = await defaultGroup.addMember(
+            auth,
+            user.toJSON()
+            // Transaction is handled internally by addMember if needed via auth
+          );
```

Comment:
It could be nice to add transaction support to those ? otherwise a group can be created and left alone if creation fails. (and we have a unique constraint on name, which would be an issue to recreate the same agent)